api4/context.go

```diff
@@ -641,3 +641,26 @@ func (c *Context) RequireActionId() *Context {
 	}
 	return c
 }
+
+func (c *Context) RequireRoleId() *Context {
+	if c.Err != nil {
+		return c
+	}
+
+	if len(c.Params.RoleId) != 26 {
```

Comment:
I think I'd rather do that separately in one go everywhere, otherwise we're going to have some places where an ID length is controlled by constant and some places where it's hard coded to 26 (as it is everywhere in the codebase at the moment). If this is going to become a constant I'd rather separately change everything all at once.

Reply:
I didn't know this was everywhere so I suppose it's OK to continue.